ons_alpha/jinja2/templates/pages/topics/topic_page.html

```diff
@@ -1 +1,47 @@
 {% extends "templates/pages/index_page.html" %}
+
+{% from "components/panel/_macro.njk" import onsPanel %}
+
+{% block main %}
+    {% block summary %}
+        <div class="ons-grid ons-js-toc-container ons-u-ml-no ons-u-mt-s">
+            <div class="ons-grid__col ons-col-8@m ons-col-12@s ons-u-p-no@xxs@m">
+                <section id="summary">
+                    <h2>{{ ("Summary")}}</h2>
```

Comment:
Suggestion: If the intention is to make this title translatable, it needs to call the `_` method, not just wrap the string in brackets:

```suggestion
                    <h2>{{ _("Summary")}}</h2>
```